README.md

```diff
@@ -1,6 +1,22 @@
 # dscigametrics
 
-Computing Google Analytics Metrics!
+`dscigametrics` is a package that could help business owners to analyze and visualize Google Analytics metrics!
+
+It provides insights of four metrics:
+1. Ratio of new to returning visitors: measures the ratio of new users to returning users of each campaign in certain period. 
+2. Conversion rate: measures the percentage of users who complete a specific desired action of each campaign in certain period.
+3. Total transaction revenue: measures total transaction revenue of each campaign in certain period.
+4. Average transaction revenue: measures average transaction revenue of each campaign in certain period.
+
+It includes four functions:
+1. `function_1` summarises general performance of campaign based on four metrics.
```

Comment:
I think these function names need to be updated. For example, function_1 is now compute_metrics